### PR TITLE
[snap] drop packaged libssh

### PR DESCRIPTION
**Pull request by Saviq**
We build our own.

### CODE REVIEW

**Comment by Saviq** [comment]
bors try

**Review by townsend2010 — COMMENTED** [review]
I still see `libssh.so.4.8.4` in the snap.  I don't think it breaks anything, but if we are doing this exercise, it should go away.  That said, it may not be trivial to get Snapcraft to remove it, so not going to block this.

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
How about now?

bors try

**Comment by Saviq** [comment]
Just for a confirmation on libvirt:

```plain
$ sudo multipass get local.driver
libvirt
$ multipass list
Name                    State             IPv4             Image
primary                 Running           192.168.122.18   Ubuntu 20.04 LTS
```

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -126,7 +133,6 @@ parts:
     - libqt5x11extras5
     - libssl1.1
     - libxml2
-    - libvirt0
```
Comment (townsend2010) [inline snap/snapcraft.yaml:129]: I think we need a:
```yaml
    after:
      - libvirt
```

for the `multipass` part, right?

**Review by townsend2010 — COMMENTED** [review]
Inline question below.

File: snap/snapcraft.yaml (hunk 1 above)
Comment (Saviq) [reply snap/snapcraft.yaml:129]: No, this is for staging. The `build-packages:` above make it so `libvirt0` is available during the build anyway.

Comment (townsend2010) [reply snap/snapcraft.yaml:129]: Oh, duh, right :+1:

**Review by townsend2010 — APPROVED** [review]
Ok, great, thanks!

bors merge

**Pull request merged**